vulntotal/datasources/vulnerablecode.py

```diff
@@ -85,3 +85,26 @@ def parse_advisory(fetched_advisory) -> VendorData:
     return VendorData(
         aliases=aliases, affected_versions=affected_versions, fixed_versions=fixed_versions
     )
+
+
+class VCIOTokenError(Exception):
+    pass
+
+
+def fetch_vulnerablecode_query(url: str, payload: dict):
+    load_dotenv()
```

Comment:
Any particular reason to use `load_dotenv()` here, we have accessed the environment variables without this as well in past for example here https://github.com/nexB/vulnerablecode/blob/main/vulnerabilities/utils.py#L238 ?

Reply:
VulnTotal is not a Django app, it cannot access the env variable loaded by Django. Hence the need to use `load_dotenv()`